spyder/plugins/variableexplorer/widgets/collectionsdelegate.py

```diff
@@ -425,6 +423,8 @@ def createEditor(self, parent, option, index):
                 self.old_obj = value.copy()
             except AttributeError:
                 self.old_obj = copy.deepcopy(value)
+            except Exception:
+                value = None
```

Comment:
Please remove these two lines or we won't be able to display an error message when trying to create an editor for a variable.